src/pages/RepoPage/BundlesTab/BundleContent/BundleContent.spec.tsx

```diff
@@ -113,66 +172,129 @@ describe('BundleContent', () => {
+        it('renders the error banner', async () => {
+          setup({ isBundleError: true, flagValue: true })
+          render(<BundleContent />, {
+            wrapper: wrapper('/gh/codecov/test-repo/bundles/main/test-bundle'),
+          })
 
-  describe('when the bundle type is not BundleAnalysisReport', () => {
-    it('renders the error banner', async () => {
-      setup({ isBundleError: true })
-      render(<BundleContent />, { wrapper })
+          const bannerHeader = await screen.findByText(/Missing Head Report/)
+          expect(bannerHeader).toBeInTheDocument()
 
-      const bannerHeader = await screen.findByText(/Missing Head Report/)
-      expect(bannerHeader).toBeInTheDocument()
+          const bannerMessage = await screen.findByText(
+            'Unable to compare commits because the head of the pull request did not upload a bundle stats file.'
+          )
+          expect(bannerMessage).toBeInTheDocument()
+        })
 
-      const bannerMessage = await screen.findByText(
-        'Unable to compare commits because the head of the pull request did not upload a bundle stats file.'
-      )
-      expect(bannerMessage).toBeInTheDocument()
+        it('renders the empty table', async () => {
+          setup({ isBundleError: true, flagValue: true })
+          render(<BundleContent />, {
+            wrapper: wrapper('/gh/codecov/test-repo/bundles/main/test-bundle'),
+          })
+
+          const dashes = await screen.findAllByText('-')
+          expect(dashes).toHaveLength(4)
+        })
+      })
     })
 
-    it('renders the empty table', async () => {
-      setup({ isBundleError: true })
-      render(<BundleContent />, { wrapper })
+    describe('flag is off', () => {
+      describe('when the bundle type is BundleAnalysisReport', () => {
+        it('renders the bundle table', async () => {
+          setup({})
+          render(<BundleContent />, { wrapper: wrapper() })
 
-      const dashes = await screen.findAllByText('-')
-      expect(dashes).toHaveLength(3)
+          const bundleName = await screen.findByText(/bundle1/)
+          expect(bundleName).toBeInTheDocument()
+
+          const bundleSize = await screen.findByText(/50B/)
+          expect(bundleSize).toBeInTheDocument()
+
+          const bundleLoadTime = await screen.findByText(/100s/)
+          expect(bundleLoadTime).toBeInTheDocument()
+        })
+      })
+
+      describe('when the bundle type is not BundleAnalysisReport', () => {
+        it('renders the error banner', async () => {
+          setup({ isBundleError: true })
+          render(<BundleContent />, { wrapper: wrapper() })
+
+          const bannerHeader = await screen.findByText(/Missing Head Report/)
+          expect(bannerHeader).toBeInTheDocument()
+
+          const bannerMessage = await screen.findByText(
+            'Unable to compare commits because the head of the pull request did not upload a bundle stats file.'
```

Comment:
This would only be expected to happen on the PR that adds bundle uploading, right?